### DISCUSSION TITLE
Some Issues

### DISCUSSION THREAD

**Discussion by Darkauu** [JetBrains/intellij-micropython#242] [Q&A] [answered]
Hello, I pass some time experimenting with Micropython, proving some sensors, verifying some common errors, etc, and something that occurs me when use some sensors like the "dht11", "dht22" or those ones that uses the library of "onewire", is that Micropython doesn't recognizes the necessary libraries, in this case "dht" and "onewire" for example. It marks me in red the library when import it and sometimes it can work, but not all the time.

Other issue is that now I update Pycharm with its last actualization, the REPL of Micropython doesn't show anything, I mean, it is totally in blank, and my microcontroller can't execute the code. Also, when I tried to hard reboot it, the led, that normally lights up for a second, doesn't light on. It makes me think that it could be because of my own microcontroller, but I'm not sure.

Thanks for your patience.

**Comment by gabemorris12** [comment]
I have also noticed the issue with the REPL not working with the latest version of PyCharm. If you've already updated your existing PyCharm installation to the latest version, you can download [toolbox](https://www.jetbrains.com/toolbox-app/) app. Go to the tools tab ---> three dots ---> other versions and install 2023.1.4.

**Reply by johnp22** [reply]
Also noticed this problem. I have backed out to PyCharm 2023.1.4 and the MicroPython REPL works fine.

**Reply by elmot** [reply]
Please try NOVA version, see #330

**Comment by randomsmith9** [comment]
will the repl issue affect intellij also? (Above mentions PYCharm)
My board will flash, then last line output is "Soft reboot" and no cursor ">>>"
and if I close that flash/repl window and click on the M icon, the "REPL" window is blank

**Reply by elmot** [reply]
Please try NOVA version, see https://github.com/JetBrains/intellij-micropython/discussions/330

**Comment by bradcar** [comment, marked as answer]
I had the same problem using Pycharm with my Raspberry Pi. Pico, then I found this site: https://micropython-stubs.readthedocs.io/en/main/packages.html#mp-packages

In the terminal in Pycharm I did the pip install of the latest, which on 25-October-2024 is: 
pip install -U micropython-rp2-stubs==1.23.0.*
and
pip install -U micropython-rp2-rpi_pico-stubs==1.23.0.*

...then the unresolved warnings went away.  ...sorry if I misunderstood your issue.
